content/posts/2025/2025-01-23-ballot-sc83/index.md

```diff
@@ -0,0 +1,122 @@
+---
+aliases:
+- /2025/01/23/ballot-sc-83-Winter-2024-2025-Cleanup-Ballot
+author: Wayne Thayer
+date: 2025-01-23
+tags:
+- Ballot
+- Server Certificates
+title: 'Ballot SC-083v3: Winter 2024-2025 Cleanup Ballot'
```

Comment:
I think the WG agreed to use the format SC083v3 (without the -). Can you please confirm? That's the format I used to change all existing ballots.

Reply:
That is also my understanding. However, I was going with the actual formatting the proposer used in the voting period email for this ballot. I have now removed the dash.